pkg/telegram/telegram.go

```diff
@@ -104,13 +121,20 @@ func parseTelegramRequest(r *http.Request) (*Update, error) {
 	return &update, nil
 }
 
-func sanitize(s string) string {
-	if len(s) >= lenStartCommand {
-		if s[:lenStartCommand] == startCommand {
-			s = s[lenStartCommand:]
+// returns the term that wants to be searched or
+// an string that specifies the expected input
+func sanitize(s string) (string, error) {
+	if len(s) >= lenSearchCommand {
+		if s[:lenSearchCommand] == searchCommand {
+			s = s[lenSearchCommand:]
+		} else {
+			return "You must enter /search {languague}", fmt.Errorf("Invalid value")
 		}
+
+	} else {
+		return "You must enter /search {languague}", fmt.Errorf("Invalid value")
```

Comment:
```suggestion
		} else {
			return "", errors.New("invalid value: you must enter /search {languague}")
		}

	} else {
		return "", errors.New("invalid value: you must enter /search {languague}")
```

When it comes to the error handling convention, if the error is different than nil, it's safe to ignore the other responses and act accordingly. Nobody expects to receive meaningful responses when a function errors out. Also, don't use `fmt.Errof` if you don't do anything on the strings, go with `errors.New` in such cases.